src/cli.ts

```diff
@@ -37,6 +60,23 @@ function buildParser(args, callback) {
         callback();
       },
     )
+    .command(
+      "search [query]",
+      "Perform a search of users, tags and places",
+      {},
+      async (handleArgs) => {
+        const logger = getLogger(handleArgs);
+        const options = getOptions(handleArgs, logger);
+        if (!handleArgs["query"]) {
+          throw new Error("query required");
+        }
+        const search = createApi("search", handleArgs["query"], options);
+        const result = await search.get();
+        process.stdout.write("\n");
+        process.stdout.write(JSON.stringify(result, null, 2));
```

Comment:
Needs an additional `\n` in sh/bash shells to put prompt on its own line.

```suggestion
        process.stdout.write(JSON.stringify(result, null, 2));
        process.stdout.write("\n");
```